spec/models/order_spec.rb

```diff
@@ -100,7 +100,7 @@
       end
     end
 
-    context 'with authorised_by_id' do
+    context 'with submitted_by_id' do
```

Comment:
You missed the change on Line 147 i.e ` context "without authorised_by_id" ` it should be ` context "without submitted_by_id" `.

Reply:
Thanks @abulsayyad123. I will correct it and will send for review to you again. :)